<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>239</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="minimumSize">
      <size>
       <width>241</width>
       <height>192</height>
      </size>
     </property>
     <property name="plainText">
      <string> # p+112Cd elastic scattering at 27.9 MeV 	  
 22.       0.548     0.044
 26.       0.475     0.024
 30.       0.481     0.014
 38.       0.447     0.009
 50.       0.144     0.004
 66.       0.499     0.010
 70.       0.248     0.005
 86.       0.463     0.014
 90.       0.485     0.015
 106.      0.087     0.003
 110.      0.135     0.004
 130.      0.161     0.005</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Type of Data and error</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_TYPE">
       <property name="maximumSize">
        <size>
         <width>209</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>angular distribution</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>double differential</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_LAB">
       <property name="maximumSize">
        <size>
         <width>209</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <item>
        <property name="text">
         <string>frame: Lab</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>frame: CM</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_IDIR_ISCALE">
       <property name="maximumSize">
        <size>
         <width>209</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>unit: ratio</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>unit: fm^2/sr</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>unit: barn/sr</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>unit: mb/sr</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>unit: micro-b/sr</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_ABSERR">
       <property name="maximumSize">
        <size>
         <width>209</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>error : absolute</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>error : relative </string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButton_clear">
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
